index.js

```diff
@@ -68,7 +68,7 @@ function exit() {
 }
 
 globals.setImmediate(function () {
-	var numberOfTests = runner.tests.concurrent.length + runner.tests.serial.length;
+	var numberOfTests = runner.tests.tests.concurrent.length + runner.tests.tests.serial.length;
```

Comment:
`tests.tests` looks kinda weird. Maybe the test collection should have a `.size` getter property (like a `Map`) or something?